### PR TITLE
Ability to delete a post.

### PR DESCRIPTION
**Pull request by pifantastic**

### CODE REVIEW

File: src/vole/vole.go [hunk 1]
```diff
@@ -5,11 +5,12 @@ import (
...
   "flag"
   "github.com/vole/web"
-  osuser "os/user"
-  "path"
   "io/ioutil"
   "lib/config"
   "lib/store"
+  osuser "os/user"
+  "path"
+  "sort"
```
Comment (marktheunissen) [inline src/vole/vole.go]: Imported and not used.

**Comment by marktheunissen** [comment]
Nice work, the basics work well. Some ideas:
- Can we only show the delete button on my posts?
- Perhaps a confirmation before deleting?

**Comment by pifantastic** [comment]
I'm keen on using `confirm()`. Anything else seems to be an over-complication.

**Comment by marktheunissen** [comment]
This is great thanks man you are the best. :cake: :man_with_turban:

**Pull request merged**